kuma/static/styles/includes/_vars.scss

```diff
@@ -122,20 +122,16 @@ $diff-font-family: $code-inline-font-family;
 /*
 sizes
 ====================================================================== */
-$h1-font-size: 50px;
-$h2-font-size: 38px;
-$h3-font-size: 24px;
+$h1-font-size: 48px;
+$h2-font-size: 24px;
```

Comment:
It is a seriously big jump from `48px` to `24px` with nothing in between. It kinda feels like there is going to be many situations where one is going to apply custom CSS to a lot of `h2` elements because `24px` feels to small, `48px` to big and, semantically you cannot use another `h1` anyhow.